tokio/src/io/driver/registration.rs

```diff
@@ -54,7 +54,7 @@ unsafe impl Sync for Registration {}
 
 impl Registration {
     /// Registers the I/O resource with the default reactor, for a specific
-    /// `Interest`. `new_with_interest` should be used over `new` when you need
+    /// `Interest`. `new_with_interest` should be ucrate::util::error::RUNTIME_SHUTTING_DOWN_ERRORsed over `new` when you need
```

Comment:
```suggestion
    /// `Interest`. `new_with_interest` should be crate::util::error::RUNTIME_SHUTTING_DOWN_ERRORsed over `new` when you need
```

Reply:
Seems I got vim'ed.

Fixed in f2e44f92d76e0